ios/FluentUI.Demo/FluentUI.Demo/Demos/AvatarGroupViewDemoController.swift

```diff
@@ -15,17 +15,44 @@ class AvatarGroupViewDemoController: DemoController {
         reloadAvatarViews()
     }
 
-    private lazy var incrementBadgeButton: MSFButton = {
-        return createButton(title: "+", action: #selector(incrementBadgeNumbers))
+    private lazy var incrementBadgeButton: MSFButtonVnext = {
+        return createButton(title: "+", action: { [weak self] _ in
```

Comment:
Can we follow a pattern of 

```swift
let button = createButton(title: "Set", action: { [weak self] _ in
        guard let strongSelf = self else {
                return
            }
           strongSelf. incrementBadgeNumbers() // replace with corresponding selectors
  })
```
This help us preserve GitHub history along with code preserving code modules. 
I see this is done is some cases and not the others. Any specific reason?

Reply:
So, these methods were not being used anywhere else and they were selectors just because the old button requires it.
I think it's worth cleaning up if there's no good reason to have them for the sake of simpler code and readability.